ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/value-display/value-display.tsx

```diff
@@ -50,11 +51,25 @@ type PermitSimulationValueDisplayParams = {
 
   /** The tokenId for NFT */
   tokenId?: string;
+
+  /** True if value is being credited to wallet */
+  credit?: boolean;
+
+  /** True if value is being debited to wallet */
+  debit?: boolean;
```

Comment:
If there is only one value, won't this two properties always alternate, in which case could we have a single `isDebit`?

Reply:
A value may be none of debit and credit, thus is it more clear to have 2 fields. These are not complete mutually exclusive.

Reply:
Could the third state be handled using the `undefined` value of the field? Is that more idiomatic to JS?